IDR/notebook/scripts/nbconverted/1.describe_screens.py

```diff
@@ -140,15 +165,65 @@ def describe_screen(screen_id):
                 # Clean channels value and separate into stain and stain target
                 stain = set()
                 stain_target = set()
-                for entry in channels.split(";"):
-                    temp_list = entry.split(":")
-                    stain.add(temp_list[0])
-                    stain_target.add(temp_list[1])
+                for channel in channels.split(";"):
+                    # For channel entries with 'stain:target' format
```

Comment:
It's fine as it is here (for now), but I suspect you'll probably want to abstract out this separation functionality into a separate `standardize_utils.py` file. I bet you'll want to use this function over and over again, in multiple places (and probably for different metadata categories!)

Reply:
@gwaybio Yes, I plan on implementing this in production!